_data/whitelist.yml

```diff
@@ -159,6 +179,8 @@ projects:
   sponsor: Sponsored by the IBM responsibility initiative
   cpu: 'yes'
   gpu: 'no'
+  gdpr: 'yes'
+  gdpr-enable-steps: https://www.worldcommunitygrid.org/ms/viewDataSharing.action
```

Comment:
Side note: if anyone has a better link for steps on world community grid, can you let me know? Right now I'm just linking directly to the setting which may be a bit confusing